aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -196,6 +196,12 @@ pub trait TDelayedFieldView {
         delayed_write_set_keys: &HashSet<Self::Identifier>,
         skip: &HashSet<Self::ResourceKey>,
     ) -> Result<BTreeMap<Self::ResourceKey, (Self::ResourceValue, Arc<MoveTypeLayout>)>, PanicError>;
+
+    fn get_group_reads_needing_exchange(
```

Comment:
let's add a comment of what the result means - i.e. that tuple returned is MetadataOp and size

Reply:
Done.